library/threading.c

```diff
@@ -49,7 +55,7 @@ static void threading_mutex_free_pthread( mbedtls_threading_mutex_t *mutex )
 
 static int threading_mutex_lock_pthread( mbedtls_threading_mutex_t *mutex )
 {
-    if( mutex == NULL || ! mutex->is_valid )
+    if( mutex == NULL || !is_mutex_valid(mutex) )
```

Comment:
style: add space after `(` and before `)`

Reply:
Fixing.